iPSL/Electrical/Wind/WindGenerator.mo

```diff
@@ -48,7 +48,7 @@ equation
           visible=true,
           origin={-0.1255,0},
           extent={{-99.8745,-96.8769},{99.8745,96.877}},
-          fileName="modelica://iPSL/../Ressources/WindGenerator.png")}),
+          fileName="modelica://iPSL/../Resources/WindGenerator.png")}),
```

Comment:
Sorry to spoil the fun (as usual ;-) ) but the Resources belong **inside** the package. Otherwise package manager would not be able to ship them correctly. So you should move `Resources` one level down under `iPSL/Resources`.